src/models/cb_admin.ts

```diff
@@ -69,6 +70,26 @@ export const CbAdmins: CbAdminCollection = {
     });
   },
 
+  async addTemporaryWithRole (client, cb) {
+    const [latestTempCbAdmin] = await client('user_account')
+      .select('*')
+      .where('email', 'like', 'welcome-%@twine-together.com')
+      .orderBy('created_at', 'desc')
```

Comment:
what happens if we sort by `email`? Wouldn't the biggest number then be first/last?

Reply:
Numbers would be seen as individual characters, so will only work up to 9.

Reply:
@eliascodes happy for me to resolve this?